superx/models/__init__.py

```diff
@@ -1,6 +1,11 @@
 from sqlalchemy import Integer, Column, Text, Boolean, Float
+<<<<<<< HEAD
```

Comment:
This happens because of a conflict. You have to modify the file to how you want it (this removes the conflict) and then add it to the repo again before commiting

Reply:
not entirey sure what you mean or what to do

Reply:
Modify the file so it only has the lines that you want and then commit it again